nix/ocaml.nix

```diff
@@ -277,35 +276,6 @@ let
 
       devnet = wrapMina self.devnet-pkg { };
 
-      berkeley-migration-pkg = self.mina-dev.overrideAttrs (s: {
```

Comment:
one reason we separated these out in the debs was overlapping package conflicts. but I imagine that's simply not a problem with nix.

the only requirement is then to ~always disambiguate which thing you want packages from? vs adding to user environment.

Reply:
Yeah, it was in this form before because different `DUNE_PROFILE` was used, but this is not the case anymore.